extensions/amp-subscriptions/0.1/test/test-amp-subscriptions.js

```diff
@@ -1441,4 +1441,43 @@ describes.fakeWin('AmpSubscriptions', {amp: true}, (env) => {
       ).to.eventually.equal(undefined);
     });
   });
+
+  describe('maybeRenderDialogForSelectedPlatform', () => {
+    it('should select and activate platform if not free and not embedded in the viewer', async () => {
+      const selectAndActivatePlatformStub = env.sandbox.stub(
+        subscriptionService,
+        'selectAndActivatePlatform_'
```

Comment:
We should never stub private methods, these tests only test 5 lines of your new method, but don't test that any platform got activated. You can refer to these tests if needed:

https://github.com/ampproject/amphtml/blob/a6546523dace584b5bb353e004df8b5481a8c051/extensions/amp-subscriptions/0.1/test/test-amp-subscriptions.js#L384